languagetool-language-modules/en/src/main/java/org/languagetool/language/English.java

```diff
@@ -73,7 +73,7 @@ public List<Rule> load(@NotNull String path) throws IOException {
           List<Rule> rules = new ArrayList<>();
           PatternRuleLoader loader = new PatternRuleLoader();
           try (InputStream is = JLanguageTool.getDataBroker().getAsStream(path)) {
-            rules.addAll(loader.getRules(is, path));
+            rules.addAll(loader.getRules(is, path, new English()));
```

Comment:
Better use this instead of creating a new instance?

Reply:
Not possible because here "this" is "CacheLoader". Anyway, I guess we should be using, whenever it's possible, the static version of the language from `Languages.getLanguageForShortCode("en")`.